config.json

```
@@ -1091,6 +1091,20 @@
           "control_flow_loops",
           "structs"
         ]
+      },
+      {
+        "slug": "rotational-cipher",
+        "name": "Rotational Cipher",
+        "uuid": "3adbc85c-7e7e-4122-a37d-c843196bc8af",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2,
```

How about a 4? I know this isn't actually difficult, but I'm redoing the track ordering and rated everything that uses *alloc as at least a 4 so that people don't run into memory management early on in the track.

```suggestion
        "difficulty": 4,
```

---

Never mind, I'll need to merge this into my branch anyway, I'll figure out a good place to stick it at that point.